section3/reflection.md

```diff
@@ -1,15 +1,40 @@
+- I can, at times, run myself a little ragged when I can't solve a problem. I may fall into the 'maybe I'm not cut out for this.' When that happens I try to take a step back. I grab a bite to eat, go outside, exercise, take a shower, or take a 20 minute nap. If I am working with other people, I tend to be able to be able to feed off of that energy and push through, especially knowing that we are all in this together. All of that to say, I can fall out of the growth mindset, but quickly try to get myself back to it.
 
 1. Think of a web or mobile application you use regularly. Where is an example of a place that the developers probably use Arrays? Explain.
+- For sure, they are commonly used on shopping sites to capture search lists, purchases, and also for capturing the songs that you 'like' on music platforms. The site can then use this data to figure out what things to present to the user that they may like based on their prior selections.
 
 1. In your own words, explain what an Array method is.
 
+- An array method is used to manipulate or to calculate items that are held within that array. Array methods normally start with the variable name, a period, then the method. for var `foods`, removing the last item from the array is the `.pop` method, so that would look like this `foods.pop`.
+
 1. What is the purpose of a loop? When we would use a loop in conjunction with array vs. without?
+- The purpose of the loop is to rid ourselves of time-consuming work in order to be more efficient and productive elsewhere. Why spend the (you estimate the hours/days/years here) into something that it takes the computer almost no time to do.
 
 1. If you had to teach a 5 year old about loops, how would you explain it?
+- A loop is just like it sounds. It goes around and around and around until you tell it to stop. First, we will tell the computer where to start. So let's tell the computer to start at 0, because that is where a computer naturally starts. This is the initializer. Then we want to tell it when to stop. Let's have it stop at 10. This is the condition. Last, we have to tell it how to increment, or how we want it to count. In this case we will have it count by one. Then we tell it what to do and that could simply be to print out a phrase. It will then print that phrase 10 times and then it will stop.
 
-1. What questions do you still have about Arrays and loops?
\ No newline at end of file
+1. What questions do you still have about
+- Arrays and loops? I feel pretty confident. I felt confident in this section, but I keep coming back to (and you all made it very clear) the understanding that `=` does not equal `===`. I've been hung up two times on this and I'm hoping to learn my lesson very VERY soon. :)
```

Comment:
See if [this](https://www.guru99.com/difference-equality-strict-operator-javascript.html#:~:text=KEY%20DIFFERENCES%3A&text=%3D%20is%20called%20as%20assignment%20operator,same%20for%20the%20two%20variables.) helps (I researched "different equals in javascript")